Plugin/Source/Processors/Loss_Effects/FIRFilter.h

```diff
@@ -5,7 +5,7 @@
 #if JUCE_MAC || JUCE_IOS
 #define Point CarbonDummyPointName
 #define Component CarbonDummyCompName
-#include <Accelerate/Accelerate.h>
+//#include <Accelerate/Accelerate.h>
```

Comment:
I think I will need some help on fixing this error. With the new chowdsp_utils commit I think using `Point` is ambiguous. I figure there's probably an easy way to fix this, I just couldn't think of one.

![Screen Shot 2022-11-16 at 1 47 16 PM](https://user-images.githubusercontent.com/43660096/202301479-8ead8800-c7bf-498d-b35d-f88c2f7a8daf.png)

Reply:
Hmm, that's annoying... I think the fix is to get rid of the `#define`s here.

Reply:
Mmm that seems to produce a different set of errors?

![Screen Shot 2022-11-17 at 9 26 04 PM](https://user-images.githubusercontent.com/43660096/202627180-86340d24-3218-48b5-b5aa-afb689470ced.png)

Reply:
@jatinchowdhury18 Might need a hand with this if you have are able, i think that might be the last thing with the PR.